submissions/serializers.py

```diff
@@ -57,6 +57,29 @@ class TeamSubmissionSerializer(serializers.ModelSerializer):
     submitted_at = DateTimeField(format=None, required=False)
     created_at = DateTimeField(source='created', format=None, required=False)
     attempt_number = IntegerField(min_value=0)
+    # Prevent Django Rest Framework from converting the answer (dict or str)
+    # to a string.
+    answer = serializers.SerializerMethodField()
+
+    def get_answer(self, obj):  # pylint: disable=unused-argument
+        answer = self.context.get("answer")
+        return answer
+
+    def validate_answer(self, value):
+        """
```

Comment:
I was gonna say that we already do this check at the individual submisisons level, but i think I actually like putting this here.